test/pod_annotations_capability.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+function run_pod() {
+	crictl runp "$TESTDATA"/sandbox_config.json
+}
+
+function delete_pod() {
+	id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	crictl stopp "$id"
+	crictl rmp "$id"
+}
+
+@test "single cni plugin with pod annotations capability enabled" {
+	enable_capability=true
+	cni_plugin_log_path="$TESTDIR/cni.log"
+
+	start_crio
+	prepare_cni_plugin "$cni_plugin_log_path" $enable_capability
```

Comment:
it's not clear to me, why are you ordering these this way? are you testing cri-o's reacting to the dropped in cni config or would it make sense to switch them?